server/routes/objects.js

```diff
@@ -124,32 +122,30 @@ const handleUpload = async (
             Data: {
               Total_Files: obj.total_files,
               Uploaded_Files: obj.curr_count,
-              format: format,
             },
           });
         }
         if (obj.curr_count === obj.total_files) {
-          await file_uploaded(user, fileId);
+          log.info(
+            `Upload to MinIO Completed ${fileName}, ${socket_user_map[socket_id]}`,
```

Comment:
Similar to line 52, ensure that `sockets[socket_id]` is defined before calling `emitMessage` to prevent runtime errors.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*